porcupine/textutils.py

```diff
@@ -790,6 +790,8 @@ def create_passive_text_widget(parent: tkinter.Widget, **kwargs: Any) -> tkinter
     kwargs.setdefault("relief", "flat")
     kwargs.setdefault("wrap", "word")
     kwargs.setdefault("state", "disabled")
+    kwargs.setdefault("takefocus", True)
+    kwargs.setdefault("highlightthickness", 0)
```

Comment:
I think setting `higlightthickness=0` and `takefocus=True` in the same text widget isn't great. You can select the widget by pressing Tab or Shift+Tab several times or by clicking it, but there's no visual indicator for it being selected. Whether it is selected or not is important for the user, because you can press Ctrl+A only in a selected widget, and you want to see where you are when navigating widgets by pressing Tab.

A better solution might be to delete these two lines:

```suggestion
```

So now, text widgets don't get focus by default, but we can add `takefocus=True` to select which widgets should be focused, and get the default visible border when focused.

`takefocus=True` is good for:
- The run output widget, so that you can copy/paste the output
- Substitutions in the "Run command" dialog, so that you can copy/paste them (#822)

`takefocus=True` is bad for:
- The two text widgets in "Run command" dialog that show the command after substituting it. I want to jump from the first entry to the second with a single Tab key press.
- The about dialog. There's no need to copy/paste its text, and a highlighted border would just look weird.

Reply:
yeah, that's a fair point i'll make modifications